### DISCUSSION TITLE
Can I run the Android apps within the Linux chroot installed through Termux ? Linux installed within Termux allows to import the USB devices exported by USB/IP server installed on Linux ?

### DISCUSSION THREAD

**Discussion by Marietto2008** [termux/termux-app#4217] [General]
Hello to everyone.

I'm going to install xfce and xorg in Termux on Android X86 version 7.1.1.

I don't know if I will be able,but I want to try. This is the tutorial that I've found and that I will try to follow :

https://mrs-th.medium.com/transform-your-android-device-into-a-linux-desktop-110a3d084ac6

But before to start,I would like to know from you,if,once did,if I will be able to run the android apps,since,I presume,Linux and xfce will run inside a chroot and I also presume that I will not be able to launch the Android apps from there.

I want also to install VirtualHere Server on Linux and VirtualHere client on Linux,too...installed through Termux...because I want to export one USB device where my mouse is connected to be able to use it in Android...

Do you think that the USB device imported through Termux will be available only inside the chroot or not even on the whole Android ?

Basically I have an old version of Android,version Phoenix OS,vers. 7.1.1 that I have virtualed in a VM created with the bhyve hypervisor that I can't use because my mouse does not work.

This is because bhyve does not support the individual USB devices and I can't use a PCI to USB controller,because NetFlix crashes if it detects it. 

So,I'm thinking to use the USB/IP technology. 

Please explain if it can work as I want.
